app/export.go

```diff
@@ -115,40 +123,65 @@ func (app *QuasarApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs
 	height := ctx.BlockHeight()
 	ctx = ctx.WithBlockHeight(0)
 
+	// reinitialize all validators (v0.46 version)
```

Comment:
do we need this commented part?

Reply:
Added it just in case to keep it as a reminder to check back on it later on.